templates/docs/examples/patterns/search-and-filter/_script.js

```diff
@@ -0,0 +1,27 @@
+/**
+  Toggles visibility of filter panel.
+  @param {HTMLElement} panel Filter panel to show or hide.
+*/
+function togglePanel(panel) {
+  if (panel) {
+    if (panel.getAttribute('aria-hidden') === 'false') {
+      panel.setAttribute('aria-hidden', 'true');
+    } else {
+      panel.setAttribute('aria-hidden', 'false');
+    }
+  }
+}
+
+// Add click handler for clicks on elements with aria-controls
+const patterns = document.querySelectorAll('.p-search-and-filter');
```

Comment:
We need these scripts to work in IE, so please use `var` instead of const/let, and cast the result of `querySelectorAll` to an array (`[].slice.call(document.querySelectorAll("...")`) to make sure `forEach` is available.

Reply:
Done.